### PR TITLE
Refactor into an opt-in structure

### PR DESCRIPTION
**Pull request by sarahsturgeon**

### CODE REVIEW

File: lua/autorun/client/cl_cfc_performance.lua [hunk 1]
```diff
@@ -25,63 +25,55 @@ local function removeHooks()
 end
 
 local function runCommands()
-    --RunConsoleCommand("cl_threaded_bone_setup", "1")
-    --RunConsoleCommand("cl_threaded_client_leaf_system", "1")
-    --RunConsoleCommand("gmod_mcore_test", "1") -- It was suggested to set this value to -2 for some reason
-    RunConsoleCommand("mat_queue_mode", "-1")
+    RunConsoleCommand("cl_threaded_bone_setup", "1")
+    RunConsoleCommand("cl_threaded_client_leaf_system", "1")
+    RunConsoleCommand("gmod_mcore_test", "1") -- It was suggested to set this value to -2 for some reason
```
Comment (justinmuskopf) [inline lua/autorun/client/cl_cfc_performance.lua:30]: Is this comment for the `RunConsoleCommand` below?

**Review by justinmuskopf — APPROVED** [review]
Approved, small thing in a comment

File: lua/autorun/client/cl_cfc_performance.lua (hunk 1 above)
Comment (sarahsturgeon) [reply lua/autorun/client/cl_cfc_performance.lua:30]: It isn't, actually. Another developer told me to set this to `-2` but I couldn't find any documentation that suggested that made any kind of sense. So 🤷‍♂

File: lua/autorun/client/cl_cfc_performance.lua [hunk 2]
```diff
@@ -25,63 +25,55 @@ local function removeHooks()
...
+    removeHooks()
+    runCommands()
+    disableWidgets()
 end
 
-local function handleWaiterTimeout()
-    print( "[CFC Enhancements] CFC Waiter timed out! Enhancements won't be loaded." )
-end
+local function warnEnable( ply, message, source )
+    local log = source == "chat" and ply:ChatPrint or print
```
Comment (plally) [inline lua/autorun/client/cl_cfc_performance.lua]:
```suggestion
    local log = source == "chat" and ply.ChatPrint or print
```
`:` is only for calling methods

**Review by plally — CHANGES_REQUESTED** [review]

File: lua/autorun/client/cl_cfc_performance.lua (hunk 2 above)
Comment (plally) [reply lua/autorun/client/cl_cfc_performance.lua]: other than this the commands seem to work

Comment (sarahsturgeon) [reply lua/autorun/client/cl_cfc_performance.lua]: Good catch!

**Pull request merged**